lib/fluent/plugin/parser_json.rb

```diff
@@ -38,7 +38,7 @@ def configure(conf)
         begin
           raise LoadError unless @json_parser == 'oj'
           require 'oj'
-          Oj.default_options = {bigdecimal_load: :float, mode: :strict}
+          Oj.default_options = {bigdecimal_load: :float, mode: :compat}
```

Comment:
This default options are different from one in `formatter_json`.
You must make a common Oj default options as a constant, and share it in everywhere.